www/js/controllers.js

```diff
@@ -25,6 +25,48 @@ module.run(function($ionicPlatform) {
 	});
 });
 
+var LoginAlerts = function () {};
+
+LoginAlerts.Helper_ErrorAlert = function ($ionicPopup, message) {
+	var alertPopup = $ionicPopup.alert(
+	{
+		title: '<span class="bold error">Error!</span>',
+		template: message
+	});
+};
+
+LoginAlerts.Helper_SuccessAlert = function ($ionicPopup, message) {
+	var alertPopup = $ionicPopup.alert(
+	{
+		title: '<span class="bold success">Success!</span>',
+		template: message
+	});
+};
+
+LoginAlerts.Error_AlreadyLoggedIn = function ($ionicPopup) {
```

Comment:
are you sure we should keep these in the class? seems like these are one off messages that can be placed in the respective controllers. It maybe useful to have convenience methods in the respective controllers and not this class for reuse....